curriculum/challenges/english/25-front-end-development/lab-contact-form/66f26c32ec6f90df01a44f60.md

```diff
@@ -0,0 +1,458 @@
+---
+id: 66f26c32ec6f90df01a44f60
+title: Design a Contact Form
+challengeType: 14
+dashedName: design-a-contact-form
```

Comment:
You shouldn't change this.

Changing the name to design was only for the title. the dashed name should stay the same as the others like this
` "dashedName": "lab-contact-form",`

Reply:
I have used the dashed name as `design-a-business-card` in the [business card lab](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/25-front-end-development/lab-business-card/6690e10ebe2181212abc9652.md) as well.

The other projects are using `build-a-x` convention in the challenge file instead of `lab-x`

So should keep `design-a-contact-form` as it is or update the the business lab project as well along with contact form lab to `lab-x`?

Reply:
oh I see, then we can leave it as is